src/sql/finalize.sql

```diff
@@ -140,6 +140,20 @@ CREATE OPERATOR - (
     RIGHTARG = veci8
 );
 
+CREATE OPERATOR * (
```

Comment:
Will we add this operator for vecf16 and veci8?

Reply:
If needed, I can add these.